tests/bin/prune-search-index_test.py

```diff
@@ -1,18 +1,30 @@
 from tests.functional_test_base import FunctionalTestBase
-from shelf.bulk_update.utils import run_search_prune
+from shelf.bucket_update.utils import prune_search_index
```

Comment:
The name of this file should be with underscores. I believe it is required that way. I thought python would not load it otherwise. Did you validate that it is running?